database/migrations/2024_11_28_000002_add_user_id_to_estudiantes_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('estudiantes', function (Blueprint $table) {
+            $table->unsignedBigInteger('user_id')->nullable()
+                ->nullable()->after('ciclo');
```

Comment:
Has repetido el método `nullable()`. Lo que me tendréis que explicar Pablo y tú es ¿cómo os podéis haber cometido este mismo error.